### ISSUE TITLE
Mosaic Tile for titles

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#155]
Theme the Mosaic Rich Text tile for use as a full width title - as in "Testimonials", "Featured Projects", "We Believe in ...", etc. The normal WYSIWYG styles will be used to determine text size and left/right/center placement.

**Comment by alecpm** [comment]
There should be tile and row styles installed by Tesserae that handle this use case for any type of tile.

**Comment by skleinfeldt** [comment]
Do I need to do something to select them, or is that something you would do as part of theming the homepage? It wasn't clear to me whether I should use the Mosaic Format dropdown for anything.

**Comment by alecpm** [comment]
You should see options for formatting tiles and rows un the Mosaic Format dropdown. Hopefully those do what they say they do, if not we can look into them.

**Comment by skleinfeldt** [comment]
The options on the Format menu are:

Tile:
Center content
Padding right
Padding bottom
Light Color Text
Float left
Float right
(Remove format)

Row:
Fluid
Dark background
(Remove format)

I put "Dark background" on the "Testimonials" row, so you can see that. https://staging.jazkarta.com/mosaic-homepage (It doesn't bleed to the edge like I requested on #154.) "Light Color Text" does make all text white, so that could be useful. "Center content" can be done in the WYSIWYG for rich text, but OK. It wasn't clear to me how the padding and float options would be useful.

Are these the formatting options that you were thinking of that tesserae installs?

In any case, this ticket may be a no-op as the normal full row rich text tile is probably fine. I just wanted to document all the types of tiles I'll need to use, in case they need any theming love.

**Comment by alecpm** [comment]
Sounds like I need to reinstall tesserae to get those registry entries in place.

**Comment by skleinfeldt** [comment]
I tried to implement a full width blue background under a title using the tesserae Background Image tile, per instructions on https://github.com/jazkarta/jazkarta.com/issues/154#issuecomment-529025665. You can see the results on https://staging.jazkarta.com/mosaic-homepage, it is the blank area just above the "Testimonials" header. I set the text to "Testimonials" (H2), color to #205c90, Display Size to Banner, left image blank, and then clicked Save. The color and text values are not saved.

**Comment by skleinfeldt** [comment]
Correction: when I saved the layout the color and text became visible, it just didn't show while editing, and the values I had set didn't show on the tile's edit form until after the layout was saved. So that's kind of funky, but it works.

However, having done this I can see that the Background Image tile is not going to be the right way to do this because it is taking its dimensions from one of the pre-defined image crops. What is needed is to set a background color under a rich text tile, and it will also be necessary to set the text color. Similar to the Mosaic format menu's Dark Background setting, but with the ability to set text and background colors, and maybe control bleeding to the edge.

I'll close this and create a more appropriate ticket.
